zerver/webhooks/insping/view.py

```diff
@@ -0,0 +1,42 @@
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, \
+    api_key_only_webhook_view
+from zerver.lib.validator import check_dict, check_string
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from typing import Dict, Any, Iterable, Optional, Text
+
+import time
+
+
+@api_key_only_webhook_view('Insping')
+@has_request_variables
+def api_insping_webhook(request: HttpRequest, user_profile: UserProfile,
+                        payload: Dict[str, Iterable[Dict[str, Any]]] = REQ(
+                            argument_type='body'),
+                        stream: Text = REQ(default='test'),
+                        topic: Text = REQ(default='insping')) -> HttpResponse:
+
```

Comment:
Okay, again this is only minor but this could be formatted for better readability:
```python
def api_insping_webhook(
        request: HttpRequest, user_profile: UserProfile,
        payload: Dict[str, Iterable[Dict[str, Any]]]=REQ(argument_type='body'),
        stream: Text=REQ(default='test'),
        topic: Text=REQ(default='insping')
) -> HttpResponse:
```

Also the spaces around `=` are probably not PEP8 compliant, so probably good to fix those. :)

Reply:
the line `payload: Dict[str, Iterable[Dict[str, Any]]]=REQ(argument_type='body'),` is over 79 chars and I remembered being told to wrap it to 79 chars, so I guess I'll do this:
```
payload: Dict[str, Iterable[Dict[str, Any]]]\
        =REQ(argument_type='body'),
```